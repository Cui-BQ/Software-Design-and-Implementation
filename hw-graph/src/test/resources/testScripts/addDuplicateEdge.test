# Test addEdge duplicate edge

# Create a graph
CreateGraph graph1

# Add 2 nodes
AddNode graph1 n1
AddNode graph1 n2

# Add different edges between n1 and n2.
AddEdge graph1 n1 n2 e1
AddEdge graph1 n1 n2 e2

# Add duplicate edges between n1 and n2.
AddEdge graph1 n1 n2 e1
AddEdge graph1 n1 n2 e2