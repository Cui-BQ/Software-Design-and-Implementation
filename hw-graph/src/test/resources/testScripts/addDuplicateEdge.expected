# Test addEdge duplicate edge

# Create a graph
created graph graph1

# Add 2 nodes
added node n1 to graph1
added node n2 to graph1

# Add different edges between n1 and n2.
added edge e1 from n1 to n2 in graph1
added edge e2 from n1 to n2 in graph1

# Add duplicate edges between n1 and n2.
failed to add edge e1 from n1 to n2 in graph1
failed to add edge e2 from n1 to n2 in graph1